### DISCUSSION TITLE
Can't validate values from an array attribute in livewire component

### DISCUSSION THREAD

**Discussion by ehsantalkhouncheh** [livewire/livewire#3091] [7. Bugs (archived)]
### Description
i got this error when i try validate in livewire :

Livewire\Exceptions\CannotBindToModelDataWithoutValidationRuleException
Cannot bind property [pagedata.utmParameters.utm_source] without a validation rule present in the [$rules] array on Livewire 
component:
and this is my validation rule
 'pagedata.utmParameters.*' => 'nullable|string',
### Stripped-down, copy-pastable code snippets

**Context**
- Livewire version: [e.g. 2.3.5]
- Laravel version: [e.g. 8.20.1]
- Alpine version: [e.g. 2.3.0]
- Browser: [e.g. Chrome, Safari]

**Comment by joshhanley** [comment]
@ehsantalkhouncheh is there a reason why you are trying to use * on the end of your rule?

The wildcard is normally used in the middle of a rule denoting you have an array of the same thing. I don't think it will work for trying to whitelist all the properties of an array. 

You will need to put the full array property name in like `'pagedata.utmParameters.utm_source' => 'nullable|string',`

Hope this helps!
